tests/test_git.py

```diff
@@ -78,6 +78,13 @@ def test_get_loc_changed(
     results = {}
 
     for label, repo_path in entropy_repository_paths.items():
+
+        # Check that repo_path in the output is the same as the input
+        # (so long as we know the path is consistent and not temp,
```

Comment:
```suggestion
        # (so long as we know the path is consistent and not "temp",
```

True? If not, then "a temp file" perhaps?

Reply:
oh, also need to close parenthesis

Reply:
Thanks, yes, I think this is true.